src/openpgp.js

```diff
@@ -45,21 +45,22 @@ import util from './util';
  * @param {Array<Object>} [options.subkeys=a single encryption subkey] - Options for each subkey e.g. `[{sign: true, passphrase: '123'}]`
  *                                             default to main key options, except for `sign` parameter that defaults to false, and indicates whether the subkey should sign rather than encrypt
  * @param {Object} [options.config] - Custom configuration settings to overwrite those in [config]{@link module:config}
+ * @param {String} symmetric - (optional) symmetric algorithm for aead/cmac keys
  * @returns {Promise<Object>} The generated key object in the form:
  *                                     { key:PrivateKey, privateKeyArmored:String, publicKeyArmored:String, revocationCertificate:String }
  * @async
  * @static
  */
-export async function generateKey({ userIDs = [], passphrase = "", type = "ecc", rsaBits = 4096, curve = "curve25519", keyExpirationTime = 0, date = new Date(), subkeys = [{}], config }) {
+export async function generateKey({ userIDs = [], passphrase = "", type = "ecc", rsaBits = 4096, curve = "curve25519", keyExpirationTime = 0, date = new Date(), subkeys = [{}], config, symmetricHash = "sha256", symmetricCipher = "aes256" }) {
   config = { ...defaultConfig, ...config };
   userIDs = toArray(userIDs);
   if (userIDs.length === 0) {
     throw new Error('UserIDs are required for key generation');
   }
-  if (type === "rsa" && rsaBits < config.minRSABits) {
-    throw new Error(`rsaBits should be at least ${config.minRSABits}, got: ${rsaBits}`);
+  if (type === "rsa" && rsaBits < config.minRsaBits) {
+    throw new Error(`rsaBits should be at least ${config.minRsaBits}, got: ${rsaBits}`);
```

Comment:
This is a rebasing error, I think

Reply:
What is the error exactly? Should these lines be removed altogether?

Reply:
The patch is changing `minRSABits` to `minRsaBits`, undoing a change that was made somewhere along the lines in v5; these lines should not be changed here.